cmd/clusterctl/cmd/init.go

```diff
@@ -56,11 +56,8 @@ var initCmd = &cobra.Command{
 		The management cluster must be an existing Kubernetes cluster, make sure
 		to have enough privileges to install the desired components.
 
-		Use 'clusterctl config repositories' to get a list of available providers; if necessary, edit
-		$XDG_CONFIG_HOME/cluster-api/clusterctl.yaml file to add new provider or to customize existing ones.
-
-		Some providers require environment variables to be set before running clusterctl init.
-		Refer to the provider documentation, or use 'clusterctl config provider [name]' to get a list of required variables.
```

Comment:
I think rather than just deleting this comment, we want to update the document to suggest an alternative method. It seems in the linked issue there was a workaround that gives you the correct variables right?

Reply:
The alternative command is already documented and the earlier command was not removed.
`Use 'clusterctl config repositories' to get a list of available providers; if necessary, edit
$XDG_CONFIG_HOME/cluster-api/clusterctl.yaml file to add new provider or to customize existing ones.`